tools/cpp/CROSSTOOL

```diff
@@ -900,6 +904,345 @@ toolchain {
   }
 }
 
+toolchain {
+  toolchain_identifier: "vc_14_0_x86_32"
```

Comment:
@mhlopko , @katre : This toolchain is almost the same as "vc_14_0_x64", the only differences are lines like 921 and 1013. Is it possible to somehow merge them and just override things in this `toolchain` rule?

Same question goes for `tools/cpp/CROSSTOOL.tpl`.

Reply:
The only way to get rid of duplicity is to generate the crosstool and build file (before the build, or in a skylark repository). There is no "reuse" mechanism built in into the cc toolchain.

Reply:
Too bad. Then I guess this is fine.

Reply:
I was able to get rid of duplicity in `tools/cpp/CROSSTOOL.tpl` (see the upcoming commit). In `CROSSTOOL`, I guess, that is okay for bootstrapping code.